documentation/docs/scenarios/changeManagement.md

```diff
@@ -0,0 +1,343 @@
+# Change Request Management and Continuous Delivery in Hybrid Scenarios
+
+Combine both ABAP and non-ABAP sources and transport them through a test landscape to a productive landscape.
+
+### Prerequisites
+
+You have downloaded the Change Management Client 2.0.0 or a compatible version. See [Maven Central Repository](http://central.maven.org/maven2/com/sap/devops/cmclient/dist.cli/).
+
+### Procedure
+
+1. Check if in SAP Solution Manager, there is a change document in status "in development". See [Check for a Change Document in Status "In Development" (`checkChangeInDevelopment`)](#check-for-a-change-document-in-status-in-development-checkchangeindevelopment).
+2. Create a transport request for a change document in SAP Solution Manager. See [(Optional) Create a Transport Request (`transportRequestCreate`)](#optional-create-a-transport-request-transportrequestcreate).
+3. (Optional) Upload a file to your transport request for a change document in SAP Solution Manager. See [Upload a File to Your Transport Request (`transportRequestUploadFile`)](#upload-a-file-to-a-transport-request-transportrequestuploadfile).
+4. Release your transport request for a change document in SAP Solution Manager. See [(Optional) Release a Transport Request (`transportRequestRelease`)](#optional-release-a-transport-request-transportrequestrelease).
+
+
+## Check for a Change Document in Status "In Development" (`checkChangeInDevelopment`)
```

Comment:
Applies only for the SOLMAN use case. For CTS there is no change document. CTS usecase only expects transport requests.